### ISSUE TITLE
OSError: [Errno 22] Invalid argument

### ISSUE THREAD

**Issue by monovertex** [Galooshi/sublime-import-js#19]
Hi, I'm getting the error in the title when trying to run Fix Imports. 

It says it's coming from `import-js.py`, line 156, which is the stdin flush.

importjs works in CLI mode without any issue.

I'm on Windows 10. Not sure what any other information to provide right now. Any ideas?

**Comment by trotzig** [comment]
Hi @monovertex, sorry for the late response. I haven't seen this issue before. Are you getting this consistently or just every now and then?

**Comment by monovertex** [comment]
Don't worry @trotzig. After I opened the issue I also encountered it on Linux and also got it to work on Windows.

I think it happens when the `importjsd` process is blocked or killed. I experienced the issue when the command failed because I had some syntax errors in the importjs config, but also saw it when, in a larger project, the process hanged because of a large number of files (see https://github.com/Galooshi/import-js/issues/510) and I had to kill it, because it blocked Sublime.

Basically, I think the plugin doesn't know to re-start the process if it's no longer up.
